resources/js/Image/JP2Image.js

```diff
@@ -92,6 +95,92 @@ var JP2Image = Class.extend(
         this._onChange();
     },
 
+    /**
+     * Compares the image metadata's timestamp to the current observation time
+     * and alerts the user if the distance is larger than a predefined
+     * threshold.
+     * @param {getClosestImageResponse} metadata
+     */
+    _notifyIfStaleImage: function (metadata) {
+        // Get the image's timestamp and the current observation time
+        let imageDate = Date.parseUTCDate(metadata.date);
+        let obstTime = helioviewer.timeControls.getDate();
+        // Get the time difference between the two times in seconds
+        let delta = Math.abs(imageDate.getTime() - obstTime.getTime()) / 1000;
+        // Get the preset threshold in seconds
+        let threshold = Helioviewer.userSettings.get("thresholds.obstime");
```

Comment:
This seems like a thing for Config, not for user settings,

Reply:
You're right, moving it to config

Reply:
Moved to serverSettings